sklearn/model_selection/_split.py

```diff
@@ -3,13 +3,15 @@
 functions to split the data based on a preset strategy.
 """
 
-# Author: Alexandre Gramfort <alexandre.gramfort@inria.fr>,
-#         Gael Varoquaux <gael.varoquaux@normalesup.org>,
+# Author: Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#         Gael Varoquaux <gael.varoquaux@normalesup.org>
 #         Olivier Grisel <olivier.grisel@ensta.org>
 #         Raghav RV <rvraghav93@gmail.com>
+#         Leandro Hermida <hermidal@cs.umd.edu>
```

Comment:
I'm confused. Is the implementation here the one by Leandro?

Reply:
It is a continuation of his [PR](https://github.com/scikit-learn/scikit-learn/pull/15239), so I left this as is.

Reply:
I'm glad my initial implementation was of use, please also add yourself and others who contributed!

Reply:
Actually that early implementation in #15239 I haven't used anymore, this is the other implementation I wrote up and use all the time https://github.com/scikit-learn/scikit-learn/issues/13621#issuecomment-600894432.  If you built from this thank you for including my name.